modules/network-observability-updating-migrating.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/installing-operators.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-updating-migrating_{context}"]
+= Migrating removed stored versions of the FlowCollector CRD
+
+Network Observability Operator version 1.6 removes the old and deprecated `v1alpha1` version of the `FlowCollector` API. If you previously installed this version on your cluster, it might still be referenced in the `storedVersion` of the `FlowCollector` CRD, even if it is removed from the etcd store, which blocks the upgrade process. These references need to be manually removed.
+
+There are two options to remove stored versions:
+
+. Use the Storage Version Migrator Operator.
+. Reinstall the Network Observability Operator, which ensures that the installation is in a clean state. 
```

Comment:
The "which" reads as referencing "Operator" but I think is supposed to reference reinstalling. So:
```suggestion
. Reinstall the Network Observability Operator. Reinstalling the Operator ensures that the installation is in a clean state. 
```

(you could probably do this another way too, just what came to mind)

Reply:
I went with 
"Reinstall the Network Observability Operator, ensuring that the installation is in a clean state.